public/stylesheets/vat.css

```diff
@@ -65,3 +65,13 @@ html[data-useragent*='MSIE 10.0'] .flex-container.tile {
 .bottom-padding {
   padding-bottom: 0.25em;
 }
+
```

Comment:
Please iterate version of file (forces browsers to not use cached version). i.e. vat-v1.css